otel-integration/k8s-helm/gke-autopilot-values.yaml

```diff
@@ -0,0 +1,503 @@
+global:
```

Comment:
Optional: I wonder if we could limit the parameters only those that are extra / changed in comparison to standard `values.yaml`, similar to how we do it for CRD and Windows "special" values file, since the values get merged during installation with defaults.

That would limit the scope and need for changes when we adjust main `values.yaml`, i.e. we would prevent having to duplicate the configuration changes in both `values.yaml` and `gke-autopilot-values.yaml`.